.github/workflows/ci.yml

```diff
@@ -30,3 +30,5 @@ jobs:
             ${{ runner.os }}-go-
       - name: Run tests
         run: go test -v ./...
+        env:
+          CI: 'true'
```

Comment:
This is unnecessary, already always defined by github https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables